apps/schools/models/students.py

```diff
@@ -41,13 +44,13 @@ class StudentGroup(models.Model):
     section = models.CharField(
         max_length=10,
         choices=Alpha_list,
-        blank=True,
+        blank=True, #Shivangi Check if this should be there!
         null=True,
-        default=''
+        default='' #Shivangi, should there be No Section? Or just null?
```

Comment:
If you want to have an empty CharField (read that as "if you want to have an empty string") in the DB, then you should always explicitly use an `empty string` to indicate that. From the Django docs:

```
Avoid using null on string-based fields such as CharField and TextField unless you have an excellent reason. If a string-based field has null=True, that means it has two possible values for “no data”: NULL, and the empty string. In most cases, it’s redundant to have two possible values for “no data;” Django convention is to use the empty string, not NULL.
```

The question however would be whether a section should be allowed to be empty at all.